core/base/Dockerfile

```diff
@@ -12,7 +12,19 @@ ARG MAILU_GID=1000
 RUN set -euxo pipefail \
   ; addgroup -Sg ${MAILU_GID} mailu \
   ; adduser -Sg ${MAILU_UID} -G mailu -h /app -g "mailu app" -s /bin/bash mailu \
-  ; apk add --no-cache bash ca-certificates curl python3 tzdata
+  ; apk add --no-cache bash ca-certificates curl python3 tzdata \
+  ; machine="$(uname -m)" \
+  ; if [[ "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ]] \
+  ; then \
+      echo "Installing hardened-malloc" \
+  ;   apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing hardened-malloc \
```

Comment:
Can you add a --no-cache here?

Reply:
Sure, that's 9b2f018be615400ed78befdcdd6649e8b815fa8d